docker/Dockerfile

```
@@ -48,6 +48,11 @@ RUN \
     os_inherit enabled true  && \
     openstack-config --set /etc/keystone/keystone.conf \
     database connection mysql+pymysql://keystone:keystone@$DB_HOST/keystone && \
+        # Set log configuration
```

```suggestion
    # Set log configuration
```

(Trying to fix indentation)